spec/functional/resource/user/useradd_spec.rb

```diff
@@ -645,6 +645,15 @@ def lock_user_account
             expect(@error).to be_a(Mixlib::ShellOut::ShellCommandFailed)
             expect(@error.message).to include("Cannot unlock the password")
           end
+        elsif %w{rhel}.include?(OHAI_SYSTEM["platform_family"]) &&
+              OHAI_SYSTEM["platform_version"].to_f = 6.8
```

Comment:
@schisamo The rhel7 on s390x does not return the error.  So I matched it to the platform version we are running on s390x